code/frameworks/preact-vite/README.md

```diff
@@ -0,0 +1,48 @@
+# Storybook for Preact <!-- omit in toc -->
+
+## Requirements
+
+- [Preact](https://preactjs.com/) >= 10.x
+- [Storybook](https://storybook.js.org/) >= 7.x
+
+## Getting Started
+
+### In a project without Storybook
+
+Follow the prompts after running this command in your Next.js project's root directory:
+
+```bash
+npx storybook@next init
+```
+
+[More on getting started with Storybook](https://storybook.js.org/docs/preact/get-started/install)
+
+### In a project with Storybook
+
+This framework is designed to work with Storybook 7. If you’re not already using v7, upgrade with this command:
+
+```bash
+npx storybook@next upgrade --prerelease
+```
+
+#### Manual migration
+
+Install the framework:
+
+```bash
+yarn add --dev @storybook/preact-vite@next
+```
+
+Update your `main.js` to change the framework property:
+
+```js
+// .storybook/main.js
+module.exports = {
+  // ...
+  framework: {
+    // name: '@storybook/react-webpack5', // Remove this
+    name: '@storybook/preact-vite', // Add this
```

Comment:
Will people necessarily be coming from `react-webpack5`?  Maybe it's better to just show the final framework?  Also, it can simplify to this:

```
framework: '@storybook/preact-vite'
```

Reply:
Yeah. I think I forgot to replace that with `@storybook/preact-webpack` since that exists, but this makes more sense given that we don't know the starting state.